Continued working on sec_paper

christianneurohr opened the pull request:
For more details see git commit.

Commit:
- Started to write Intersections (espc. main theorem)
- Moved macros from DEInt to macros.tex
- Minor updates to the rest

Commit:
Merge branch 'master' of https://github.com/pascalmolin/hcperiods

Commit:
- Continued writing Section Intersections
- Updated macros
- Added several graphics

Commit:
Merge branch 'master' of https://github.com/pascalmolin/hcperiods

Conflicts:
	doc/sec_paper/DoubleExponentialIntegration.tex
	doc/sec_paper/macros.tex

Commit:
- Updated macros again

Commit:
- First draft of Section Intersections finished (added tikz_pic_5)
  - May add more cases to the proof
- Wrote some comments and correted typos in Sections DEI/AJM
- Minor updates to Sections SEC/StratPM
- Started writing Section CAJ
- Added Pascals thesis as bib-entry

pascalmolin commented:
Great, you just wrongly added some bib and backup files (bbl, blg, .backup), you can remove them.
Also there is at least a dupplicate unicode macro (varphi).

Thanks for the comments on Abel Jacobi, I think the exposition is a bit too condensed, I plan to introduce more details on the jacobian, as you pointed.

Commit:
- Removed unnecessary files

Commit:
- Added paragraph about winding number to CompAspcts
- Small corrections in SEC
- Minor changes to macros
- Started writing CompAJ.tex
